packages/react/src/components/Checkbox/Checkbox.mdx

```diff
@@ -0,0 +1,84 @@
+import { Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import { Checkbox, CheckboxSkeleton } from '../Checkbox';
+
+# Checkbox
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Skeleton state](#skeleton-state)
+- [Component API](#component-api)
+- [Feedback](#feedback)
+
+## Overview
```

Comment:
Should we include an example where we show how to do a "controlled" example? e.g.

```js
function ExampleComponent() {
  const [isChecked, setIsChecked] = useState(false);
  return <Checkbox checked={isChecked} onChange={setIsChecked} />
}
```

Reply:
Thats a good idea. Simple and clean example. I can add this.

Reply:
@aledavila was this ever added?

Reply:
@tw15egan yes it’s under overview in controlled example. Do you think this should instead be under API?

Reply:
Ah whoops, my mistake! missed that 👍 

Looks good where it is, I just wanted to make sure we can link directly to it